iam/iam-policy-for-ecs/core/README.md

```diff
@@ -19,45 +19,44 @@ locals {
       github = "user"
       name   = "Mary Lamb"
       groups = ["engineer"]
-      staging-developer-access-projects    = "metaverse:messenger"
-      staging-admin-access-projects        = "metaverse"
-      production-developer-access-projects = "metaverse:facebook"
-      production-admin-access-projects     = ""
+      project_access = {
+        developer = {
+          staging        = ["facebook", "metaverse"]
+          production     = ["facebook"]
+          sandbox        = ["facebook", "metaverse"]
+          loadtesting    = ["facebook", "metaverse"]
+        }
+        admin = {
+          sandbox = ["facebook"]
+        }
+      }
     }
   }
 }
 
 resource "aws_iam_user" "user" {
   for_each = local.users
   name     = each.value["iam"]
+
   tags = {
     name                                 = each.value["name"]
     github                               = each.value["github"]
-    staging-developer-access-projects    = try(each.value["staging-developer-access-projects"], "")
-    staging-admin-access-projects        = try(each.value["staging-admin-access-projects"], "")
-    production-developer-access-projects = try(each.value["production-developer-access-projects"], "")
-    production-admin-access-projects     = try(each.value["production-admin-access-projects"], "")
+
+    staging-developer-access-projects    = join(":", try(each.value["developer"]["staging"], []))
+    staging-admin-access-projects        = join(":", try(each.value["admin"]["staging"], []))
+    production-developer-access-projects = join(":", try(each.value["developer"]["production"], []))
+    production-admin-access-projects     = join(":", try(each.value["admin"]["production"], []))
+
+    sandbox-admin-access-projects        = join(":", try(each.value["admin"]["sandbox"], []))
   }
 }
 
 module "iam_ecs_policies" {
   source = "github.com/dbl-works/terraform//iam/iam-policy-for-ecs/core?ref=v2022.05.18"
 
-  username = <iam-user-name>
-  region   = "eu-central-1"
-
-  projects = [
-    {
-      name        = "facebook",
-      environment = "sandbox"
-      region      = "eu-central-1"
-    },
-    {
-      name        = "facebook",
-      environment = "load-testing"
-      region      = "eu-central-1"
-    },
-  ]
+  username       = local.users["gh-user"]["iam"]
```

Comment:
do we need to iterate over `local.uses` and creat that policy for each user?

Reply:
Yes. because the policy is user-specific